week3/homework/src/index.js

```diff
@@ -1,3 +1,78 @@
 'use strict';
+let express = require('express');
+let app = express();
+let uuid = require('uuid/v4')
+app.use(express.json());
+let { readFile, writeFile } = require('./todo.js');
+
+app.get('/READ', (req, res) => {
```

Comment:
Why is this `/READ` instead of `/todos`?

Also it should return read to-dos to the client through response `res` and not just write them to the console.